.gitpod.yml

```diff
@@ -9,7 +9,7 @@ tasks:
   - init: |
       rm ./renovate.json
       yarn install
-    command: yarn redwood dev
+    command: yarn rw dev --fwd="--allowed-hosts localhost --allowed-hosts $(gp url 8910 | cut -c 9-) --client-web-socket-url=$(gp url 8910 | sed -e "s/https/wss/g")/ws"
```

Comment:
@necropolina I don't have an environment to test this locally at the moment, but `--client-web-socket-url` looks like it always uses the Gitpod workspace URL, even on `localhost`. Is this what you had in mind?

Reply:
If you were hosting this locally, you'd probably just be using `yarn rw dev` either manually or in some other script, and hot reload would be working fine out of the box. These configs should only really ever need to get run when accessing the interface from somewhere other than the app's host anyways. And I don't think it'd pose a problem for non-gitpod-local development, since the `.gitpod.yml` file probably isn't being used at all for local development.

The reason I set localhost as an allowed host here is to support Gitpod's [local companion](https://www.gitpod.io/blog/local-app) which allows you to access ports exposed in the workspace's container from `localhost`. So `localhost:8910/ws` essentially points to the same place as the workspace's URL ... just going through the tunnel. 😛

Reply:
🤦 of course, LGTM 🙂!

Reply:
Although... I just went to double check and realized that the 8910 port is being applied to the workspace URL when going from localhost. Thought I'd tested that, oops! Looking for a fix for that.

Also, I'm gonna change that `sed` in the socket url to use `cut` instead, on the off chance there's an `https` in the subdomain 😛

Reply:
I think there might be a bug with webpack... even if I specifically put `:443` in the URL, it gets overwritten as `8910` when accessing from localhost via local companion. Opened an issue on the webpack repo.

Dunno if y'all would want to wait to see if anything comes from that... or, for the time being, could just add a comment stating that folks should use `yarn rw dev` if they're using local companion?

Reply:
@mikenikles I just tested this on Gitpod and it looks good! But by locally, did you mean local companion?

> Dunno if y'all would want to wait to see if anything comes from that... or, for the time being, could just add a comment stating that folks should use yarn rw dev if they're using local companion?

@necropolina I'm down to document what to do for local companion instead of waiting for a webpack fix, but if you think it'll be a quick turnaround we could wait too. But if we opt to document, maybe just a quick note in the super-outdated README would suffice?

Reply:
Apparently the issue stems from webpack not being able to establish a websocket connection over https from an http origin. One of their maintainers suggested using the `--https` flag, but that renders the interface inaccessible (seeing 502s). Not sure what the source of that is... maybe it's conflicting with something else in RW's webpack setup?

Sooo... unless that info gives y'all a bright idea, I'll just keep an eye on that issue and maybe tinker with it a bit more sometime.

In the meantime, I'd think it'd be fine to just add a comment above the command in `gitpod.yml`... I feel like people using local companion are probably also used to checking & manipulating the `gitpod.yml` file. Tho I guess I have only a sample size of me to go off of there. 😛